sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -474,6 +474,14 @@ class EthService(
     blockId
   }
 
+  private def getBlockIdByHashOrTag(nodeView: NV, tag: String): ModifierId = {
+    if (tag.length == 66 && tag.substring(0, 2) == "0x")
+      bytesToId(Converter.fromHexString(tag.substring(2)))
```

Comment:
Please use any of the other conversion tools, e.g. `ByteUtils.fromHexString`. `Converter` is supposed to be internal to the `evm` package.

Reply:
Fixed